pkg/k8sutil/constants.go

```diff
@@ -20,9 +20,15 @@ const (
 	KubeConfigEnvVar = "KUBECONFIG"
 
 	// WatchNamespaceEnvVar is the constant for env variable WATCH_NAMESPACE
-	// which is the namespace that the pod is currently running in.
+	// which is the namespace where the watch activity happens and where the pod is currently running in.
+	// this value is empty if the operator is running with clusterScope.
 	WatchNamespaceEnvVar = "WATCH_NAMESPACE"
 
+	// NamespaceEnvVar is the constant for env variable NAMESPACE
+	// which is the namespace that the pod is currently running in.
+	// this value is set only if the operator is running with clusterScope
+	NamespaceEnvVar = "NAMESPACE"
```

Comment:
Could this be renamed to `ClusterNamespaceEnvVar` as like you explain it's only set if the operator is running with cluster scope?

Reply:
Also maybe `OPERATOR_NAMESPACE` would be a better env var name as `NAMESPACE` seems very generic and might have conflicts.

Reply:
renaming done:
GetNamespace -> getOperatorNamespace
NamespaceEnvVar -> OperatorNamespace